config/prod.exs

```diff
@@ -9,6 +9,11 @@ config :logger, level: :info
 
 config :uptime, UptimeWeb.Endpoint, cache_static_manifest: "priv/static/cache_manifest.json"
 
+config :uptime,
+  supported_config_dirs: ["/etc/uptime"],
```

Comment:
I would think these paths would be hardcoded in the helm chart and docker compose configurations we provide, is there a certain installation case where the user would configure this? Seems like a way to cause confusion and do things wrong if we let a user configure this

Reply:
Not user configured, just configured by us in case we need to add a config dir in user space or somewhere else. I can hardcode them if you prefer.

Reply:
I also wanted a separate directory for local dev and testing

Reply:
ah ok, that sounds good to me then